typing/tast_iterator.ml

```diff
@@ -99,20 +117,30 @@ let structure_item sub {str_desc; str_env; _} =
   | Tstr_class list ->
       List.iter (fun (cls,_) -> sub.class_declaration sub cls) list
   | Tstr_class_type list ->
-      List.iter (fun (_, _, cltd) -> sub.class_type_declaration sub cltd) list
-  | Tstr_include incl -> include_infos (sub.module_expr sub) incl
+      List.iter (fun (_, s, cltd) ->
+        iter_loc sub s; sub.class_type_declaration sub cltd) list
+  | Tstr_include incl -> include_infos sub (sub.module_expr sub) incl
   | Tstr_open od -> sub.open_declaration sub od
   | Tstr_attribute _ -> ()
```

Comment:
Note that `attribute` and `attributes` types contain `Location.t` values in them. These need to be added to the traversal code.